maps/tether/tether_turfs.dm

```diff
@@ -36,7 +38,8 @@ VIRGO3B_TURF_CREATE(/turf/simulated/floor/reinforced)
 VIRGO3B_TURF_CREATE(/turf/simulated/floor/tiled/steel_dirty)
 /turf/simulated/floor/tiled/steel_dirty/virgo3b/New()
 	..()
-	outdoor_turfs.Add(src)
+	if(outdoors)
```

Comment:
This one is for sure redundant with above New()

Reply:
Though even if I did know, I shouldn't have done this, true.